### DISCUSSION TITLE
Email confirmations - Email Wrong - username blocked

### DISCUSSION THREAD

**Discussion by caosdp-rs** [codeigniter4/shield#632] [Q&A]
In my application, I require email confirmation. There is a scenario where the user registers in the application and makes a mistake with the email. How can they correct the wrong email after having requested registration? Even without confirming the email, the username chosen by the user remains tied to the incorrect email.
And the user cannot access another page as the shield always redirects them to `/auth/a/show`, preventing them from registering a new email

**Comment by datamweb** [comment]
The solution to this issue is the ability to edit the email in the image that has entered Active code.
I've put the code in practice, but I'm not sure if this feature should be added to the shield right now.

![Screenshot 2023-02-21 125729](https://user-images.githubusercontent.com/9530214/220304453-2b13a3ac-848b-465a-ad7a-dfa9bfa9daa8.jpg)
